utilities/test_suite/HOST/Tensor_host_audio.cpp

```diff
@@ -173,6 +195,29 @@ int main(int argc, char **argv)
             maxWallTime = std::max(maxWallTime, wallTime);
             minWallTime = std::min(minWallTime, wallTime);
             avgWallTime += wallTime;
+
+            // QA mode - verify outputs with golden outputs. Below code doesn’t run for performance tests
+            if (testType == 0)
```

Comment:
Why did we increase to 8 reference outputs for to_decibels? QA has always been 3, non_silent_region has 3 too

Reply:
@snehaa8 Could  you remove those reference outputs not needed?

Reply:
Done removed extra ones, it was 8 initially and Sampath made this change in NSR at a later point.